pkgs/by-name/tt/tt-rss/package.nix

```diff
@@ -30,13 +30,13 @@ stdenv.mkDerivation rec {
   '';
 
   passthru = {
-    inherit (nixosTests) tt-rss;
+    tests = { inherit (nixosTests) tt-rss; };
     updateScript = unstableGitUpdater { hardcodeZeroVersion = true; };
   };
 
   meta = with lib; {
     description = "Web-based news feed (RSS/Atom) aggregator";
-    license = licenses.gpl2Plus;
+    license = licenses.gpl3;
```

Comment:
```suggestion
    license = licenses.gpl3Plus;
```

Reply:
I don't see a "or later" clause in the [COPYING file](https://git.tt-rss.org/fox/tt-rss.git/tree/COPYING).

Reply:
It's in README, line 12: https://git.tt-rss.org/fox/tt-rss.git/tree/README.md

Reply:
GPLv3 is mentioned 2 times, GPLv3+ only once, weird. I've opened a [bug report](https://community.tt-rss.org/t/conflicting-gplv3-and-gplv3-and-later-licenses-in-the-project/6690) to get a definitive answer. In the meantime, I've put GPLv3+, as it's a superset of GPLv3.

Reply:
Fox from the project [answered](https://community.tt-rss.org/t/conflicting-gplv3-and-gplv3-and-later-licenses-in-the-project/6690/2?u=gileri), it should be `gpl3Plus`.